troposphere/views/web_shell.py

```diff
@@ -21,18 +22,25 @@ def _create_signature(secret, *parts):
 
 def web_shell(request):
     response = HttpResponse()
+    shell_user = request.user
+    if request.session:
+        logger.info("request.session: %s" % request.session.items())
```

Comment:
I see that this log line was already present, just under the `is_authenticated()` check so - nevermind me!